<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaffmanagementWindow</class>
 <widget class="QMainWindow" name="StaffmanagementWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:
qlineargradient(spread:pad, x1:0.502, y1:0.0611111, x2:0.527155, y2:1, stop:0 rgba(0, 255, 104, 255), stop:0.971014 rgba(123, 255, 165, 255))
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>1021</width>
      <height>651</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="deletepushButton">
       <property name="font">
        <font>
         <family>思源宋体 CN</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="addpushButton">
       <property name="font">
        <font>
         <family>思源宋体 CN</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>增加</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="returnpushButton">
       <property name="font">
        <font>
         <family>思源宋体 CN</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>返回主界面</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="3">
      <widget class="QTableWidget" name="tableWidget">
       <property name="font">
        <font>
         <family>思源宋体 CN</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
